pkg/sync/sync_context.go

```diff
@@ -810,13 +811,27 @@ func (sc *syncContext) autoCreateNamespace(tasks syncTasks) syncTasks {
 					tasks = append(tasks, nsTask)
 				} else {
 					sc.log.WithValues("namespace", sc.namespace).Info("Namespace already exists")
```

Comment:
It seems that this log entry is a bit misleading. Maybe the message needs to be rephrased or moved elsewhere?

Reply:
Ah right, that should go in the line under the `liveObj` nil check?

Reply:
correct!